tests/writer.js

```diff
@@ -24,7 +24,8 @@ require([
                     onReady: function () {
                         done();
                     }
-                }
+                },
+                features: {'experimental_ui': false},
```

Comment:
I didn't do that as I thought it would serve as a good and more explicit reminder to me of what tests support the ui and which don't, and it only adds noise to this PR.

It's also easy to accidentally commit tests/options.js since we'll have experimental_ui on pretty much all the time